### PR TITLE
update get-lit scripts

### PR DESCRIPTION
**Pull request by truemedian**

### CODE REVIEW

**Review by SinisterRectus — APPROVED** [review]
Not sure why this was never accepted. Looks fine here.

**Comment by creationix** [comment]
I think I'm done moving things around, the latest version should be:

- luvi @ 2.12.0
- lit @ 3.8.4
- luvit @ 2.18.0

The Luvit release is pending review of this PR https://github.com/luvit/luvit/pull/1142

File: get-lit.ps1 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-$LUVI_VERSION = "2.10.1"
-$LIT_VERSION = "3.8.1"
+$LUVI_VERSION = "2.12.0"
+$LIT_VERSION = "3.8.2"
```
Comment (creationix) [inline get-lit.ps1:3]: Update to `"3.8.4"

File: get-lit.sh [hunk 2]
```diff
@@ -1,18 +1,19 @@
 #!/bin/sh
 set -eu
-LUVI_VERSION=${LUVI_VERSION:-2.10.1}
-LIT_VERSION=${LIT_VERSION:-3.8.1}
+LUVI_VERSION=${LUVI_VERSION:-2.12.0}
+LIT_VERSION=${LIT_VERSION:-3.8.2}
```
Comment (creationix) [inline get-lit.sh:4]: Update to `"3.8.4"`

**Review by creationix — CHANGES_REQUESTED** [review]

**Review by creationix — APPROVED** [review]

**Pull request merged**